ui/lib/password-expiration.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "ethers";
+import { useMemo } from "react";
+import { useAccount } from "./use-wagmi";
+import { useVeNationLock } from "./ve-token";
+
+const threshold = ethers.BigNumber.from(10).pow(18).mul(3).div(2);
```

Comment:
Would you just put this as `1500000000000000000` (1.5*10^18) in the `.env.local` file?

Reply:
Yep

Reply:
Actually right now we have `NEXT_PUBLIC_VENATION_REQUIRED_STAKE=2` so I would do this one in the same format (1.5) or change both to reflect the full uint256 value instead of the decimal value.